src/components/App/App.component.jsx

```diff
@@ -31,27 +33,45 @@ function App() {
   }, []);
 
   return (
-    <BrowserRouter>
-      <AuthProvider>
-        <Layout>
-          <Switch>
-            <Route exact path="/">
-              <HomePage />
-            </Route>
-            <Route exact path="/login">
-              <LoginPage />
-            </Route>
-            <Private exact path="/secret">
-              <SecretPage />
-            </Private>
-            <Route path="*">
-              <NotFound />
-            </Route>
-          </Switch>
-          <Fortune />
-        </Layout>
-      </AuthProvider>
-    </BrowserRouter>
+    <div>
+      <CssBaseline />
+      <Navbar />
+      <BrowserRouter>
+        <AuthProvider>
+          {/* <Layout> */}
+            <Switch>
+              <Route exact path="/">
+                <HomePage />
+              </Route>
+              <Route path="*">
+                <NotFound />
+              </Route>
+            </Switch>
+          {/* </Layout> */}
+        </AuthProvider>
+      </BrowserRouter>
+    </div>
+    // <BrowserRouter>
+    //   <AuthProvider>
+    //     <Layout>
+    //       <Switch>
+    //         <Route exact path="/">
+    //           <HomePage />
+    //         </Route>
+    //         <Route exact path="/login">
+    //           <LoginPage />
+    //         </Route>
+    //         <Private exact path="/secret">
+    //           <SecretPage />
+    //         </Private>
+    //         <Route path="*">
+    //           <NotFound />
+    //         </Route>
+    //       </Switch>
+    //       <Fortune />
+    //     </Layout>
+    //   </AuthProvider>
+    // </BrowserRouter>
```

Comment:
Here as well, I suggest you to remove the commented code because is noise and problematic to work on in the maintenance